variants/templates/variants/filter_result/header.html

```diff
@@ -31,6 +31,9 @@
   <th class="effect-change-protein"></th>{# effect change protein #}
   <th class="effect-change-cdna"></th>{# effect change cdna #}
   <th class="distance-to-splicesite"></th>{# distance to splicesite #}
+  {% for value in result_extra_annos_header %}
+    <th class="effect-change-extra_annos-{{ forloop.counter }}"></th>
```

Comment:
the class name is weird, this is not an effect-change thing. also, the class can probably be dropped as it not used anyway?

Reply:
ah, seems to be used in the dropdown. naming is still weird.

Reply:
https://github.com/bihealth/varfish-server/blob/7538557af9c4fa6fb43f055dc4e75a52947c7e11/varfish/static/js/state_machine.js#L417 this defines which columns have the little asc/desc sort button. please check if this still holds after your addition.

Reply:
class name changed, effect-change removed. The problem might be column 29 which now dynamically changes. Could we allow sort at that position?

Reply:
Column 29, which is the last one, comes before extra anno columns so none of them will be affected.